uint32 id
uint16 coordinate_system    # EPSG
float64 x                   # [m]
float64 y                   # [m]